packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -341,13 +423,19 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 
                 if(dataModel.haveTaskLock()) {
                     dataModel.counter.increment(1);
+
                     if (dataModel.counter.value % opsPerCycle === 0) {
+                        const maybeP = dataModel.blobFinish();
+                        if (maybeP) {
```

Comment:
this can just be `await dataModel.blobFinish()` no need for the varialbe of the if